main.py

```diff
@@ -18,4 +18,15 @@ def read_item(item_id: int, q: Union[str, None] = None):
 @app.post("/orders", response_class=PlainTextResponse)
 def get_order(order_details: order.Order):
     order.save_order(order_details)
-    return f"order saved\n"
\ No newline at end of file
+    return f"order saved\n"
+
+@app.get("/price")
+def calculate_price(miles: int,
+                    pallets: int = 0,
+                    packages: int = 0):
```

Comment:
You should add the return type as well as your function is not void and we are doing typing. It can be done like this:

```
def calculate_price(miles: int, pallets: int = 0, packages: int = 0) -> float:
```